core/domain/user_services.py

```diff
@@ -170,6 +172,16 @@ def validate(self):
         if self.role not in role_services.PARENT_ROLES:
             raise utils.ValidationError('Role %s does not exist.' % self.role)
 
+        if not isinstance(self.creator_dashboard_display_pref, basestring):
+            raise utils.ValidationError(
+                'Expected dashboard display preference to be a string, '
+                'received %s' % self.creator_dashboard_display_pref)
+        if (self.creator_dashboard_display_pref not in
+                constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.values()):
+            raise utils.ValidationError(
+                '%s is not a valid value for the dashboard display'
```

Comment:
Add a space at the end of this string (before the closing quote), otherwise it'll end up showing as "displaypreferences".

Reply:
Done